src/SymbolPaths/QSettingsWrapper.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "SymbolPaths/QSettingsWrapper.h"
+
+#include <QSettings>
+#include <filesystem>
+
+constexpr const char* kSymbolPathsSettingsKey = "symbol_directories";
+constexpr const char* kDirectoryPathKey = "directory_path";
+
+namespace orbit_symbol_paths {
+
+std::vector<std::filesystem::path> LoadPaths() {
+  QSettings settings{};
+  const int size = settings.beginReadArray(kSymbolPathsSettingsKey);
+  std::vector<std::filesystem::path> paths{};
+  paths.reserve(size);
+  for (int i = 0; i < size; ++i) {
+    settings.setArrayIndex(i);
+    std::filesystem::path new_directory{};
+    new_directory = settings.value(kDirectoryPathKey).toString().toStdString();
```

Comment:
nit: maybe inline this?
`paths.emplace_back(settings.value(kDirectoryPathKey).toString().toStdString());`

Reply:
yes! done